tests/nested/core/hotplug/task.yaml

```diff
@@ -0,0 +1,193 @@
+summary: Create ubuntu classic image, install snapd and test hotplug feature
+
+prepare: |
```

Comment:
The test looks identical to the existing one for classic, except for prepare part (unless I missed something). I wonder if it would be possible to set a global variable and simply reffer to it in prepare to do classic or core preparation?

Reply:
If thats the case then yeah, I think we should totally look unifying this, the script is rather long so the less duplication of this the better.